build/prepare-safari-release.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+
+# Automatically exit on error
+set -e
+
+CONFIG_FILE=./safari/LocalOverrides.xcconfig
+
+TAG=$(git describe --tags --abbrev=0)
+
+if [[ $(git describe --tags) != "$TAG" ]]; then
+	echo You’re ahead of the latest tag. Run:
+	echo git checkout "$TAG"
+	exit 1
+fi
```

Comment:
This is to ensure we don't get mismatched releases (e.g. "it's 22.3.3 _but_ there are 3 more bugfixes")

If you need to make small changes, I think you'll be able to:

1. check out the tag
2. make the changes
3. run this script
4. _then_ commit them